sample-windows.ini

```diff
@@ -14,9 +14,11 @@ portNumber = 8000
 remoteSymbolServer = http://symbolapi.mozilla.org:80/
 
 ; These options are unlikely to need editing by users
-maxCacheEntries = 2000000
-mruSymbolStateFile = %(TEMP)s\snappy-mru-symbols.json
-maxMRUSymbolsPersist = 10
+maxCacheEntries = 400
```

Comment:
Can you make it so that files fetched remotely (i.e. URLFetcher fetching from S3) get cached in both disk caches), but not files read remotely (i.e. PathFetcher fetching from local disk)?
This is really an improvement, so leave it for a different pull request